src/os_posix/os_fs.c

```diff
@@ -681,12 +681,19 @@ __posix_open_file(WT_FILE_SYSTEM *file_system, WT_SESSION *wt_session,
 	 * interesting.
 	 */
 	if (!pfh->direct_io && file_type == WT_FS_OPEN_FILE_TYPE_DATA) {
+		advise_flag = POSIX_FADV_NORMAL;
```

Comment:
Yes, and also `FILE_FLAG_SEQUENTIAL_SCAN`.  There is no equivalent to `POSIX_FADV_NORMAL`.  I'll fix that up.